README.md

```diff
@@ -419,9 +419,9 @@ Requirements:
 
 ### Test
```

Comment:
Do we still have automated (CI) tests for both browser and node?

Reply:
Yes, there is. `yarn test` runs all test for browser and node. `yarn test:node` and `yarn test:browser` for node and browser respectively